src/rdb_protocol/datum_stream.cc

```diff
@@ -1991,11 +1991,13 @@ std::vector<changespec_t> union_datum_stream_t::get_changespecs() {
 range_datum_stream_t::range_datum_stream_t(bool _is_infinite_range,
                                            int64_t _start,
                                            int64_t _stop,
+                                           int64_t _step,
                                            backtrace_id_t bt)
     : eager_datum_stream_t(bt),
       is_infinite_range(_is_infinite_range),
```

Comment:
Ah, didn't see your comment before I posted mine. Yea, I thought about allowing `0` too but realized the usage is a little janky since you'd have to specify the meaningless `stop` argument to get the number you want: `r.range(1, 0, { step: 0 })`. Let me know what you think.